### PR TITLE
Fix readme for upcoming hardfork

### PR DESCRIPTION
**Pull request by arnuschky**
Fixes issue #2426

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,12 +95,12 @@ Monero uses a fixed-schedule hard fork mechanism to implement new features. This
...
-| 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations  |
-| 2017-01-05        | v4                | v0.10.1                 | v0.10.2.1                   | Allow normal and RingCT transactions |
-| 2017-04-15        | v5                | v0.10.3.0               | v0.10.3.1                    | Adjusted minimum blocksize and fee algorithm      |
-| 2017-09-21        | v6                | Not determined as of 2017-03-27                | Not determined as of 2017-03-27                    | Allow only RingCT transactions      |
+| 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations |
+| 2017-01-05        | v4                | v0.10.1                | v0.10.2.1                  | Allow normal and RingCT transactions |
+| 2017-04-15        | v5                | v0.10.3.0              | v0.10.3.1                  | Adjusted minimum blocksize and fee algorithm |
+| 2017-09-21        | v6                | v0.11.0.0              | v0.11.0.0                  | Allow only RingCT transactions, default ring size is 5  |
```
Comment (hyc) [inline README.md]: The wallet default was already 5. This hardfork changes the minimum to 5.

Comment (arnuschky) [reply README.md]: That's what I meant, but expressed badly. Fixed.

**Comment by moneromooo-monero** [comment]
Looks like a duplicate of 2405.

**Comment by hyc** [comment]
Agreed, let's close this in favor of #2405 and also close issue #2426

**Comment by moneromooo-monero** [comment]
+duplicate 2405

does that work ?

**Comment by danrmiller** [comment]
+duplicate
(duplicate of #2405)

**Pull request closed without merging**